serverless/src/scaleImage/utils/sharp/resizeImage.js

```diff
@@ -0,0 +1,36 @@
+import sharp from 'sharp'
+
+/**
+ * Resize a given image to a given height and width
+ * @param {Buffer<Image>} image An image binary contained in a buffer
+ * @param {Integer} height image height
+ * @param {Integer} width image width
+ * @return {Buffer<Image>} Resized image or null
+ */
+export const resizeImage = async (image, height, width) => {
+  try {
+    const w = parseInt(width, 10)
```

Comment:
Your jsdoc comment says `height` and `width` are already integers, why are you calling parseInt?

Reply:
So in the `static/src/js/components/GranuleResults/GranuleResultsFocusedMeta.js` when we pass the height and width as URL parameters they are strings (this is when you select a specific granule and get the modal of the image on the bottom right; and when you open it up to be larger) however, when we pull out the default value from the `static.config.json` it is a number. I've updated the Jsdoc to reflect that.

Reply:
My initial thoughts were that those values are numbers, the docs say they are numbers, why are you parsing them? I'd just parse them right after reading the parameters in the handler. Why does the default value in the config matter? I would think we always specify the size we want an image in the front end code

Reply:
Yea I had removed the h and w params where we were specifying in our calls for the cases where that was equal to the defaults. What you say is more clear moved parsing to start of stack for the lambda and reverted to passing those values in from the calls to it